exe/wallet/Main.hs

```diff
@@ -100,11 +100,11 @@ active server and can be ran "offline" (e.g. 'generate mnemonic')
 
 Usage:
   cardano-wallet server [--port=INT] [--bridge-port=INT]
-  cardano-wallet generate mnemonic [--size=INT]
-  cardano-wallet list wallets [--port=INT]
-  cardano-wallet get wallet --wallet-id=STRING [--port=INT]
-  cardano-wallet create wallet --name=STRING [--address-pool-gap=INT] [--port=INT]
-  cardano-wallet delete wallet --id=STRING
+  cardano-wallet mnemonic generate [--size=INT]
+  cardano-wallet wallet list [--port=INT]
+  cardano-wallet wallet get [--port=INT] --wallet-id=STRING
+  cardano-wallet wallet create [--port=INT] --name=STRING [--address-pool-gap=INT]
+  cardano-wallet wallet delete [--port=INT] --id=STRING
```

Comment:
`--wallet-id=STRING` ?

Reply:
Yes? :sweat_smile: ?

Reply:
Yes... or `cardano-wallet wallet get [--port=INT] --id=STRING` :)

Reply:
Ah! That's what you meant :smile: ... I've just made this a positional argument actually (as discussed in the ticket), so I'll push on top of this commit.